Code/GraphMol/FileParsers/PDBParser.cpp

```diff
@@ -728,4 +733,219 @@ RWMol *PDBFileToMol(const std::string &fileName, bool sanitize, bool removeHs,
   return PDBDataStreamToMol(static_cast<std::istream *>(&ifs), sanitize,
                             removeHs, flavor, proximityBonding);
 }
+
+#ifdef HAS_GEMMI
+namespace {
+void parseMMCifFile(RWMol *&mol,
+                    const std::string &fname, bool sanitize, bool removeHs,
+                    unsigned int flavor, bool proximityBonding) {
+  PRECONDITION(fname.length(), "no filename");
+  gemmi::cif::Document doc =  gemmi::cif::read_file(fname);
+
+  gemmi::cif::Block &b = doc.sole_block();
+
+  mol = new RWMol();
+
+  gemmi::cif::Table t = b.find("_atom_site.",
+                               {"group_PDB", "type_symbol",
+                                "?label_atom_id",
+                                "?label_alt_id", "?label_seq_id",
+                                "?label_comp_id", "?label_asym_id",
+                                "?pdbx_PDB_ins_code",
+                                "?occupancy",
+                                "?B_iso_or_equiv",
+                                "?pdbx_format_charge",
+                                "?Cartn_x", "?Cartn_y", "?Cartn_z"});
+// column numbers for various variables to avoid finger trouble
+#define GROUP_PDB 0
```

Comment:
Please avoid using `#define` for constants. The preferred approach to this would be:
`constexpr size_t GROUP_PDB=0;`